waku/v2/node/wakunode2.nim

```diff
@@ -440,10 +440,19 @@ when defined(rln):
     proc validator(topic: string, message: messages.Message): Future[ValidationResult] {.async.} =
       let msg = WakuMessage.init(message.data) 
       if msg.isOk():
-        #  check the proof
-        if node.wakuRlnRelay.rlnInstance.proofVerify(msg.value().payload, msg.value().proof):
-          return ValidationResult.Accept
-      return ValidationResult.Reject
+        let 
+          wakumessage = msg.value()
+          # validate the message
+          validationRes = node.wakuRlnRelay.validateMessage(wakumessage)
+        case validationRes:
+          of Valid:
+            return ValidationResult.Accept
```

Comment:
Maybe trace log here?

Reply:
Sure, done in https://github.com/status-im/nim-waku/pull/769/commits/754e62cfb564ececa3b2a555d348372844b22db9